UI/cypress/e2e/complaints-page.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Complaints page
+
+  Scenario: Complaints page is displayed
+    Given the ‘contact us’ page is displayed 
```

Comment:
Please use the shared "a user has arrived on the contact us page" from pages.js

Reply:
Done